Recommendation on ACKing (#233)
quicwg/ops-drafts: draft-ietf-quic-applicability.md

expose an interface that allows an application layer to specify how to apply
padding.

> # ACKing on constraint links

Reviewer: ```suggestion
# ACK-only packets on constrained links
```

On network paths where there is significant asymmetry,

> the transmission of QUIC ACKs can reduce the efficient use of network

Reviewer: ```suggestion
the transmission of packets only containing ACK frames can reduce the efficient use of network
```

> capacity or transmission opportunities. This effect occurs

Reviewer: ```suggestion
capacity. This effect occurs
```
This sentence is still a bit awkward, but I'm not sure what else to suggest.

Reviewer 2: ... Radio technologies can have costs per-packet, not per byte - and a sending packet high rate impacts that... in ways for QUIC that would not happen currently with TCP. 
I suggest a new additional sentence, something like:
"On some types of link, the rate of transmission of QUIC packets carrying control information can also impact the overall link efficiency and ultimately the user experience."

> when the return capacity is significantly more constrained than the

Reviewer: I don't believe return and forward are used elsewhere, but this sentence also seems a bit redundant to the previous sentence which discusses asymmetric paths.

Reviewer 3: Maybe instead:

> The cost of sending acknowledgments - in processing cost or link utilization - could be a significant proportion of available resources if these resources are constrained.  Reducing the rate at which acknowledgments are generated can preserve these resources and improve overall performance.

Reviewer 2: This text (or similar) seems OK to me.

Is "overall" here meaning QUIC user performance  (or overall link-layer?) I think impact on performance for the QUIC user is the more significant additional take-away.

Reviewer 3: Overall was meant to be all-inclusive, which definitely includes application-relevant metrics like time to first byte, total transfer time, transfer rate, ....  It's clear that there are many ways in which this might help.

Reviewer 2: We agree. Although my question is, it is obvious to a QUIC user that there user performance can be impacted - for several decades they could have used TCP, where other mitigations exist? So can we add "(including application-relevant metrics)" or something?
